src/components/patch/PatchContainer.test.tsx

```diff
@@ -10,7 +10,19 @@ import { PatchContainer } from "./PatchContainer";
 
 describe("PatchContainer", () => {
 
-  const wrapper = enzyme.mount(<PatchContainer client={rest.EvergreenClient("", "", "", "", true)} />);
+  const checkState = jest.fn(() => {
+    wrapper.update();
+    expect(wrapper.state("expandedPatches")).toEqual({ "5d1385720305b932b1d50d01": 1 });
+    const patch = wrapper.findWhere(node => node.key() === "5d1385720305b932b1d50d01");
+    expect(patch).toHaveLength(1);
+    // tslint:disable-next-line: no-string-literal
```

Comment:
If this is scoped to the test, you can avoid this code duplication and just set the patch variable once